install/operator/go.mod

```diff
@@ -51,4 +51,6 @@ replace (
 	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
 )
 
-replace git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Comment:
Not sure about this, if we used `v0.12.0` and now we're switching to https://github.com/apache/thrift/commit/2566ecd5d999, we would [miss 68 commits](https://github.com/apache/thrift/compare/2566ecd5d999...v0.12.0).